tests/settings.py

```diff
@@ -27,6 +27,15 @@
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
 )
 
+# For debug toolbar
+if os.getenv('PES_DEBUG_TOOLBAR'):
+    INSTALLED_APPS += ('debug_toolbar',)
+    MIDDLEWARE_CLASSES += ('debug_toolbar.middleware.DebugToolbarMiddleware',)
+    INTERNAL_IPS = ['127.0.0.1']
```

Comment:
I believe we can actually get rid of this, since the next setting is basically saying to allow everyone to see the toolbar regardless of their IP address. (I now realize I need to remove the INTERNAL_IPS setting from some other settings files for the same reason)